src/main/java/uk/gov/hmcts/reform/coh/service/QuestionRoundService.java

```diff
@@ -17,13 +19,40 @@
     private QuestionRepository questionRepository;
     private QuestionStateService questionStateService;
 
+    public QuestionRoundService() {}
+
     @Autowired
     public QuestionRoundService(QuestionRepository questionRepository, QuestionStateService questionStateService) {
         this.questionRepository = questionRepository;
         this.questionStateService = questionStateService;
     }
 
-    public boolean validateQuestionRound(Question question, OnlineHearing onlineHearing) {
+    public boolean isQrValidState(Question question, OnlineHearing onlineHearing) {
+        int targetQuestionRound = question.getQuestionRound();
+        int currentRoundNumber = getCurrentQuestionRoundNumber(onlineHearing);
+
+        Optional<QuestionState> optionalQuestionState = questionStateService.retrieveQuestionStateByStateName("ISSUED");
+        if(!optionalQuestionState.isPresent()){
+            throw new NotAValidUpdateException();
+        }
+
```

Comment:
1. The mapQuestionState method actually required, so putting it in the constructor makes more sense